apps/roam/src/utils/hyde.ts

```diff
@@ -1,16 +1,18 @@
-/* eslint-disable @typescript-eslint/naming-convention */
 import { getLoggedInClient } from "./supabaseContext";
 import { Result } from "./types";
 import normalizePageTitle from "roamjs-components/queries/normalizePageTitle";
 import findDiscourseNode from "./findDiscourseNode";
 import { nextApiRoot } from "@repo/utils/execContext";
+import { DiscourseNode } from "./getDiscourseNodes";
+import getExtensionAPI from "roamjs-components/util/extensionApiContext";
 
 type ApiEmbeddingResponse = {
   data: Array<{
     embedding: number[];
   }>;
 };
 
+/* eslint-disable @typescript-eslint/naming-convention */
```

Comment:
Need to add this at the end of the function as well to close the wrapping.  Otherwise it affects the whole file (everything after the comment).